### PR TITLE
Chore/COR 1431 remove recommended advices

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary
Removed recommended advices page

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-01 at 16 23 58

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-01 at 16 23 54

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Looks good to me. Great cleanup. I've just one question in a comment. Also, I assume that this is also on backend's radar? How does it work now that the schema does not ask for measures data but it's still this in the JSONs?

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Comment by VWSCoronaDashboard18** [comment]
> Looks good to me. Great cleanup. I've just one question in a comment. Also, I assume that this is also on backend's radar? How does it work now that the schema does not ask for measures data but it's still this in the JSONs?

@VWSCoronaDashboard28 This PR doesn't include any backend data schema changes. Only Sanity configs. The page to be removed doesn't use backend data. But the sanity changes would still need a cleanup when this gets released...

**Comment by VWSCoronaDashboard28** [comment]
> > Looks good to me. Great cleanup. I've just one question in a comment. Also, I assume that this is also on backend's radar? How does it work now that the schema does not ask for measures data but it's still this in the JSONs?
> 
> @VWSCoronaDashboard28 This PR doesn't include any backend data schema changes. Only Sanity configs. The page to be removed doesn't use backend data. But the sanity changes would still need a cleanup when this gets released...

Good point. Indeed all the Schema changes are scoped to the `cms` directory. Thanks for clarifying!

File: docs/deleting-a-page-procedure.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+1. ### imports (if there are any)
+   Is the imported function/variable/type used elsewhere?
+      - Yes. only delete the import.
+      - No. Delete that file too. (Go recursively through the files meaning go to the first step of this list)
+2. ### getStaticProps
+   - Delete the sanity lokalize keys in the nl_export.json
+   - Delete the JSON schema if the proto data is not used elsewhere.
+   - Delete the Sanity schema/structure if the imported sanity content is not used elsewhere.
+
+3. ### delete the whole page file.
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply docs/deleting-a-page-procedure.md]: Yes, I tried making it more clear.
If it's okay now, please resolve!

File: packages/app/src/next-config/redirects/redirects.js [hunk 2]
```diff
@@ -127,6 +117,27 @@ async function redirects() {
...
+    {
+      source: '/veiligheidsregio/:code/geldende-adviezen',
+      destination: '/',
+      permanent: true,
+    },
+    {
+      source: '/landelijk/geldende-adviezen',
+      destination: '/',
+      permanent: true,
+    },
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/next-config/redirects/redirects.js:140]: Are these redirects already set in stone, or still up for discussion?

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Pre-approving, as the code generally looks good and the docs are clear! However, do check my one question.

File: packages/app/src/next-config/redirects/redirects.js (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/next-config/redirects/redirects.js:140]: These are the ones communicated by the communication team. I've updated the ticket comments as well.

Comment (VWSCoronaDashboard18) [reply packages/app/src/next-config/redirects/redirects.js:140]: Could you resolve this if this is an answer ti your liking?

**Pull request merged**